web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -132,13 +132,24 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
         if (values.parameter) {
             return template.apply({
                 "label": label,
-                "value": this.getMeasuredParameters(values)
+                "value": this._getMeasuredParametersAsCommaSeparatedString(values)
             });
         }
         return "";
     },
 
-    getMeasuredParameters: function(values) {
+    _getMeasuredParametersAsCommaSeparatedString: function(values) {
```

Comment:
Maybe consider calling this `_getMeasuredParametersText()` or something? It's a touch shorter and we wouldn't have to change it if we changed symbols again (we used to separate with pipes '|').